samza-core/src/main/java/org/apache/samza/util/Util.java

```diff
@@ -51,6 +51,10 @@ public static String envVarEscape(String str) {
         .replace("`", "\\`");
   }
 
+  public static String getAppEnv(Config config) {
+    return new ApplicationConfig(config).getAppEnv();
+  }
+
```

Comment:
Why do we need this if we already have the accessor in ApplicationConfig?

Reply:
That's a good question... 
If you look at this Util.java's getTaskClassVersion() function, you will find it still calls ApplicationConfig's getAppClass function. 
Not sure whether I need to add a getAppEnv() method in Util.java. What's your opinion?